app/code/Magento/SalesRule/Model/Rule/Condition/Address.php

```diff
@@ -61,6 +61,7 @@ public function __construct(
     public function loadAttributeOptions()
     {
         $attributes = [
+            'base_subtotal_with_discount' => __('Subtotal (Excl. Tax)'),
```

Comment:
Are you sure that the `base_subtotal_with_discount` is the Subtotal Excluding Tax?
Its name (and usage in the code) makes me think that it is not.
Maybe I am wrong, can you help me understand?
Thanks!

Reply:
I'm pretty sure that that's the correct subtotal to apply a coupon. I've taken it from the [collect](https://monosnap.com/file/69g8LCXETuoMUYCik9zTvmec4XBTQq ) method which is always running. All that data in any case will be in the Quote.